contracts/mixnet/src/contract.rs

```diff
@@ -180,8 +205,118 @@ pub fn query(deps: Deps, _env: Env, msg: QueryMsg) -> Result<QueryResponse, Cont
     Ok(query_res?)
 }
 
+fn get_all_mixnodes_identities(deps: &DepsMut) -> Result<Vec<IdentityKey>, ContractError> {
```

Comment:
I think those functions are only used in migration (?). Could we be more explicit about it? For example we could put it directly inside the migration, i.e.
```rust
#[entry_point]
pub fn migrate(deps: DepsMut, _env: Env, _msg: MigrateMsg) -> Result<Response, ContractError> {
   fn get_all_mixnodes_identities(deps: &DepsMut) -> Result<Vec<IdentityKey>, ContractError> {
   ...
   }

...
}

Reply:
Sure, I'll reduce their scope.